web_widget_text_markdown/README.rst

```diff
@@ -73,18 +73,7 @@ Authors
 * Alexandre Díaz
 * Komit
 * Sudokeys
-
-Contributors
-~~~~~~~~~~~~
-
-* Nicolas Jeudy <nicolas@sudokeys.com>
-* Nguyen Tan Phuc <phuc.nt@komit-consulting.com>
-* Alexandre Díaz <dev@redneboa.es>
-
-Other credits
-~~~~~~~~~~~~~
-
-* Komit https://komit-consulting.com
+* Kevin
```

Comment:
You should leave all the previous Contributors there, and add yourself underneath.

It's actually better/easier to use the OCA 'readme generator' for this, `oca-gen-addon-readme`.

So you'd

1. Look at the OCA template module and copy some example files from there such as [CONTRIBUTORS.rst](https://github.com/OCA/maintainer-tools/tree/master/template/module/readme)
2. Fill it in with the correct guys and copy it [here](https://github.com/KKamaa/web/tree/0ec75cc1a65b58dbb0b264d24e5a38d812f2eb32/web_widget_text_markdown/readme)
3. Install `oca_gen_addon_readme` on your system. [Guide](https://github.com/OCA/maintainer-tools/blob/master/README.md#installation) or just:
```
$ git clone git@github.com:OCA/maintainer-tools.git
$ pip install --user ./maintainer-tools
```

4. Run it [like this](https://github.com/OCA/maintainer-tools/blob/master/README.md#readme-generator)

Reply:
followed the steps, also added Sunflower IT.

Reply:
Almost correct: a contributor is always a person, so it should be kevin@sfit

Reply:
fixed